src/views/GovernanceTaskList/index.tsx

```diff
@@ -212,19 +236,23 @@ const GovernanceTaskList = () => {
                       direct you to go through the remaining steps.
                     </p>
                   </TaskListDescription>
-                  <Alert type="info">
-                    <span>
-                      To help with that review, someone from the IT Governance
-                      team will schedule a phone call with you and Enterprise
-                      Architecture (EA).
-                    </span>
-                    <br />
-                    <br />
-                    <span>
-                      After that phone call, the governance team will decide if
-                      you need to go through a full governance process.
-                    </span>
-                  </Alert>
+                  {['INTAKE_DRAFT', 'INTAKE_SUBMITTED'].includes(
```

Comment:
Can you leave a comment explaining why these statuses were chosen?

Reply:
Good idea, added one.